daemons/gptp/gptp_cfg.ini

```diff
@@ -31,19 +31,11 @@ syncReceiptThresh = 8
 
 [eth]
 
-# PHY delay GB TX in nanoseconds
+# PHY delay GB TX/RX in nanoseconds
```

Comment:
Thanks @christopher-s-hall. I missed this last week. I think we are good to merge as is then.